src/components/Categories/CategoryList/CategoryList.jsx

```diff
@@ -1,5 +1,26 @@
+import categoriesList from '../../../temp/categories.json';
+import css from './CategoryList.module.css';
+import sprite from '../../../../src/images/icons.svg'
+
 const CategoryList = () => {
-  return <ul>CategoryList</ul>;
+  return (
+    <ul className={css.categoriesList}>
+      {categoriesList.map(category => (
+        <li className={css.categoryItem}>
+          <div className={css.categoryLinkContainer}>
+            <a className={css.categoryLink} href="">
+              {category.name}
+            </a>
+            <a className={css.categoryArrow} href="">
```

Comment:
Link to={`recipes/?category=${category.name}`}

Reply:
Corrected